tests/x11/firefox.pm

```diff
@@ -23,9 +23,11 @@ sub run() {
 
     $self->start_firefox;
     wait_still_screen;
-    send_key_until_needlematch('firefox-top-bar-highlighted', 'alt', 4, 10);
-    send_key('h');
-    wait_still_screen;
+    # to make sure the menu can show
```

Comment:
@tinawang123 Can you please add more comments why alt/alt-v used here? I am a little confused 

alt key is used for? and alt-v is used for?